app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -35,13 +35,19 @@ import com.rob729.quiethours.Database.Profile
 import com.rob729.quiethours.Database.ProfileViewModel
 import com.rob729.quiethours.R
 import com.rob729.quiethours.databinding.FragmentMainBinding
-import com.rob729.quiethours.util.SwipeToDeleteCallback
+import com.rob729.quiethours.util.*
 
 /**
  * A simple [Fragment] subclass.
  *
  */
 class MainFragment : Fragment() {
+    companion object {
```

Comment:
What is the logic of having this companion object when already the variable in it is stored in the shared preference? 
It will be better to use the only shared preference to access and write the stored variables.

Reply:
@rob729 ok I will do so. Is everything else fine?

Reply:
Code wise looks fine. Not tested the app as of now. Make these changes and then I will test that.